### ISSUE TITLE
FOM Behaviour

### ISSUE THREAD

**Issue by cdebom** [LSSTDESC/tomo_challenge#29]
Dear @joezuntz and @EiffL. We found out that the resulting FOM_3x2 can be very different in multiple trainings.

For instance, if we use the RF example w/ 5% for training  (100 times - 5 bins) we have found in the validation

![image](https://user-images.githubusercontent.com/29743719/89658603-1f9fef00-d8a5-11ea-8a65-51ceb57a91e8.png)

Thus, one can find  10^4 as well as 6*10^4 using the same algorithm.

while the SNR_3x2 have variations lowers than 0.6 :

`![image](https://user-images.githubusercontent.com/29743719/89659827-ccc73700-d8a6-11ea-96bd-6529f0f8b1e5.png)

If, instead of using RF we add the same the redshifts in the correct bin for the validation sample we find (5 bins) FOM_3x2  lower than 10^4, depending upon some selection. 

Does this make sense for you?

**Comment by pwhatfield** [comment]
![tomo_4](https://user-images.githubusercontent.com/11774835/91448830-492fb300-e872-11ea-8769-7ca9950a68cc.png)
I have found a similar issue when varying parameters that describe the training smoothly, see figure, the FOM values jump around a fair bit...

**Comment by dkirkby** [comment]
Are you using `jax-cosmo` option to calculate the FoM?  It should have better numerical stability.

**Comment by pwhatfield** [comment]
Er probably not sorry; does tc.compute_scores just need jax-cosmo=True or something?

**Comment by EiffL** [comment]
yes, essentially :-) 
@cdebom were you using the jax-cosmo version of the metrics? Because they should be more stable

**Comment by cdebom** [comment]
Yes, it seems that this was the issue. We are using jax-cosmo now. Thanks
